<?xml version="1.0" encoding="UTF-8"?>
<testsuite xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="https://maven.apache.org/surefire/maven-surefire-plugin/xsd/surefire-test-report.xsd" name="models.EndangeredAnimalTest" time="2.683" tests="7" errors="5" skipped="0" failures="0">
  <properties>
    <property name="java.runtime.name" value="OpenJDK Runtime Environment"/>
    <property name="java.vm.version" value="11.0.8+10"/>
    <property name="sun.boot.library.path" value="/home/makena/.sdkman/candidates/java/11.0.8.hs-adpt/lib"/>
    <property name="maven.multiModuleProjectDirectory" value="/home/makena/Videos/sonnie"/>
    <property name="java.vm.vendor" value="AdoptOpenJDK"/>
    <property name="java.vendor.url" value="https://adoptopenjdk.net/"/>
    <property name="guice.disable.misplaced.annotation.check" value="true"/>
    <property name="path.separator" value=":"/>
    <property name="java.vm.name" value="OpenJDK 64-Bit Server VM"/>
    <property name="sun.os.patch.level" value="unknown"/>
    <property name="user.country" value="US"/>
    <property name="sun.java.launcher" value="SUN_STANDARD"/>
    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
    <property name="user.dir" value="/home/makena/Videos/sonnie"/>
    <property name="java.vm.compressedOopsMode" value="32-bit"/>
    <property name="java.runtime.version" value="11.0.8+10"/>
    <property name="java.awt.graphicsenv" value="sun.awt.X11GraphicsEnvironment"/>
    <property name="os.arch" value="amd64"/>
    <property name="java.io.tmpdir" value="/tmp"/>
    <property name="line.separator" value="&#10;"/>
    <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
    <property name="os.name" value="Linux"/>
    <property name="classworlds.conf" value="/usr/share/maven/bin/m2.conf"/>
    <property name="sun.jnu.encoding" value="UTF-8"/>
    <property name="java.library.path" value="/usr/java/packages/lib:/usr/lib64:/lib64:/lib:/usr/lib"/>
    <property name="maven.conf" value="/usr/share/maven/conf"/>
    <property name="jdk.debug" value="release"/>
    <property name="java.class.version" value="55.0"/>
    <property name="java.specification.name" value="Java Platform API Specification"/>
    <property name="sun.management.compiler" value="HotSpot 64-Bit Tiered Compilers"/>
    <property name="os.version" value="5.4.0-42-generic"/>
    <property name="library.jansi.path" value="/usr/share/maven/lib/jansi-native"/>
    <property name="user.home" value="/home/makena"/>
    <property name="user.timezone" value="Africa/Nairobi"/>
    <property name="java.awt.printerjob" value="sun.print.PSPrinterJob"/>
    <property name="file.encoding" value="UTF-8"/>
    <property name="java.specification.version" value="11"/>
    <property name="user.name" value="makena"/>
    <property name="java.class.path" value="/usr/share/maven/boot/plexus-classworlds-2.x.jar"/>
    <property name="java.vm.specification.version" value="11"/>
    <property name="sun.arch.data.model" value="64"/>
    <property name="sun.java.command" value="org.codehaus.plexus.classworlds.launcher.Launcher heroku:deploy"/>
    <property name="java.home" value="/home/makena/.sdkman/candidates/java/11.0.8.hs-adpt"/>
    <property name="user.language" value="en"/>
    <property name="java.specification.vendor" value="Oracle Corporation"/>
    <property name="awt.toolkit" value="sun.awt.X11.XToolkit"/>
    <property name="java.vm.info" value="mixed mode"/>
    <property name="java.version" value="11.0.8"/>
    <property name="securerandom.source" value="file:/dev/./urandom"/>
    <property name="java.vendor" value="AdoptOpenJDK"/>
    <property name="maven.home" value="/usr/share/maven"/>
    <property name="file.separator" value="/"/>
    <property name="java.version.date" value="2020-07-14"/>
    <property name="java.vendor.url.bug" value="https://github.com/AdoptOpenJDK/openjdk-support/issues"/>
    <property name="sun.io.unicode.encoding" value="UnicodeLittle"/>
    <property name="sun.cpu.endian" value="little"/>
    <property name="java.vendor.version" value="AdoptOpenJDK"/>
    <property name="sun.desktop" value="gnome"/>
    <property name="sun.cpu.isalist" value=""/>
  </properties>
  <testcase name="endangeredAnimal_instantiatesCorrectly_true" classname="models.EndangeredAnimalTest" time="0.001"/>
  <testcase name="getHealth_returnsCorrectHealthAttribute_true" classname="models.EndangeredAnimalTest" time="0"/>
  <testcase name="all_returnsAllInstancesOfEndangeredAnimal_true" classname="models.EndangeredAnimalTest" time="0.514">
    <error message="Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host &quot;196.202.208.169&quot;, user &quot;hhgxpccvuyngmn&quot;, database &quot;d5j2hvlid2h80o&quot;, SSL off" type="org.sql2o.Sql2oException">org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "196.202.208.169", user "hhgxpccvuyngmn", database "d5j2hvlid2h80o", SSL off
	at models.EndangeredAnimalTest.all_returnsAllInstancesOfEndangeredAnimal_true(EndangeredAnimalTest.java:30)
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "196.202.208.169", user "hhgxpccvuyngmn", database "d5j2hvlid2h80o", SSL off
	at models.EndangeredAnimalTest.all_returnsAllInstancesOfEndangeredAnimal_true(EndangeredAnimalTest.java:30)
</error>
  </testcase>
  <testcase name="update_updatesAgeAttribute_true" classname="models.EndangeredAnimalTest" time="0.582">
    <error message="Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host &quot;196.202.208.169&quot;, user &quot;hhgxpccvuyngmn&quot;, database &quot;d5j2hvlid2h80o&quot;, SSL off" type="org.sql2o.Sql2oException">org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "196.202.208.169", user "hhgxpccvuyngmn", database "d5j2hvlid2h80o", SSL off
	at models.EndangeredAnimalTest.update_updatesAgeAttribute_true(EndangeredAnimalTest.java:57)
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "196.202.208.169", user "hhgxpccvuyngmn", database "d5j2hvlid2h80o", SSL off
	at models.EndangeredAnimalTest.update_updatesAgeAttribute_true(EndangeredAnimalTest.java:57)
</error>
  </testcase>
  <testcase name="save_assignsIdAndSavesObjectToDatabase" classname="models.EndangeredAnimalTest" time="0.526">
    <error message="Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host &quot;196.202.208.169&quot;, user &quot;hhgxpccvuyngmn&quot;, database &quot;d5j2hvlid2h80o&quot;, SSL off" type="org.sql2o.Sql2oException">org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "196.202.208.169", user "hhgxpccvuyngmn", database "d5j2hvlid2h80o", SSL off
	at models.EndangeredAnimalTest.save_assignsIdAndSavesObjectToDatabase(EndangeredAnimalTest.java:22)
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "196.202.208.169", user "hhgxpccvuyngmn", database "d5j2hvlid2h80o", SSL off
	at models.EndangeredAnimalTest.save_assignsIdAndSavesObjectToDatabase(EndangeredAnimalTest.java:22)
</error>
  </testcase>
  <testcase name="find_returnsAnimalWithSameId_secondAnimal" classname="models.EndangeredAnimalTest" time="0.519">
    <error message="Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host &quot;196.202.208.169&quot;, user &quot;hhgxpccvuyngmn&quot;, database &quot;d5j2hvlid2h80o&quot;, SSL off" type="org.sql2o.Sql2oException">org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "196.202.208.169", user "hhgxpccvuyngmn", database "d5j2hvlid2h80o", SSL off
	at models.EndangeredAnimalTest.find_returnsAnimalWithSameId_secondAnimal(EndangeredAnimalTest.java:40)
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "196.202.208.169", user "hhgxpccvuyngmn", database "d5j2hvlid2h80o", SSL off
	at models.EndangeredAnimalTest.find_returnsAnimalWithSameId_secondAnimal(EndangeredAnimalTest.java:40)
</error>
  </testcase>
  <testcase name="update_updatesHealthAttribute_true" classname="models.EndangeredAnimalTest" time="0.539">
    <error message="Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host &quot;196.202.208.169&quot;, user &quot;hhgxpccvuyngmn&quot;, database &quot;d5j2hvlid2h80o&quot;, SSL off" type="org.sql2o.Sql2oException">org.sql2o.Sql2oException: Could not acquire a connection from DataSource - FATAL: no pg_hba.conf entry for host "196.202.208.169", user "hhgxpccvuyngmn", database "d5j2hvlid2h80o", SSL off
	at models.EndangeredAnimalTest.update_updatesHealthAttribute_true(EndangeredAnimalTest.java:49)
Caused by: org.postgresql.util.PSQLException: FATAL: no pg_hba.conf entry for host "196.202.208.169", user "hhgxpccvuyngmn", database "d5j2hvlid2h80o", SSL off
	at models.EndangeredAnimalTest.update_updatesHealthAttribute_true(EndangeredAnimalTest.java:49)
</error>
  </testcase>
</testsuite>